app/components/public/session-item.hbs

```diff
@@ -61,7 +61,7 @@
             {{/each}}
           </div>
         </div>
-        <div class="right floated five wide column pt-1" >
+        <div class="right floated five wide column" style="padding-top: .25rem !important;" >
```

Comment:
Not here

Reply:
classes r not working here also

Reply:
Let me think of it

Reply:
Try mt class

Reply:
mt classes (none of utlis classes) are not working there , and there is extra space due to padding